tests/pipelines/stable_diffusion/test_stable_diffusion_img2img.py

```diff
@@ -122,7 +137,9 @@ def test_stable_diffusion_img2img_default_case(self):
         image_slice = image[0, -3:, -3:, -1]
 
         assert image.shape == (1, 32, 32, 3)
-        expected_slice = np.array([0.4492, 0.3865, 0.4222, 0.5854, 0.5139, 0.4379, 0.4193, 0.48, 0.4218])
+        expected_slice = np.array(
+            [0.46275955, 0.3977616, 0.42548066, 0.5823421, 0.50115615, 0.43968713, 0.41080174, 0.47410887, 0.42165133]
```

Comment:
Let's not change those tests, could you maybe change the defaults of `self.get_dummy_inputs` so that the test doesn't have to be changed?

Reply:
I changed the `expected_slice` back but had to add one line to the tests to ensure the image processor won't normalize the torch tensor to `[-1,1]` so that the results match  

in the original img2img pipeline implementation, the preprocess function will only normalize pil image and leave it unchanged if image input is a torch tensor - in our test image input is a random torch tensor between `[0,1]`, it should be normalized to `[-1,1]` but it wasn't. I think for the testing purpose it is ok to leave it as it is.